odk1-src/security-privacy.rst

```diff
@@ -11,16 +11,29 @@ Security and Privacy
 
 .. _security-and-privacy:
 
-Below is an attempt to enumerate known security and privacy considerations of the ODK software.
+This document details known security and privacy considerations of the ODK software.
 
-1. License Terms
-------------------
+.. contents::
+  :local:
 
-The ODK software is released under the `Apache 2 License <http://www.apache.org/licenses/LICENSE-2.0>`_.
+.. _license:
 
-All our installers, programs, and source code are provided AS-IS with no warranty or conditions, and without any liability obligations. See the license text for details.
+License 
+-----------
 
-2. Communication Channels
+The ODK software is released under the `Apache 2 License`_.
+
+.. _Apache 2 License: http://www.apache.org/licenses/LICENSE-2.0
+
+The ODK Documentation is released under the `Creative Commons Attribution 4.0 International License <cc-by-4>`_.
```

Comment:
Instead of ODK Documentation, I'd say "All other artifacts (for example, documentation) is released under..."